seed/seed.json

```diff
@@ -134,6 +134,122 @@
                 "channel": ["NIGHTLY", "BETA"],
                 "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
             }
+        },
+        {
+            "name": "AdNotificationsDisabledOnBetaAndReleaseChannelsStudy",
+            "experiments": [
+                {
+                    "name": "AdNotifications",
+                    "probability_weight": 100,
+                    "parameters": [
+                        {
+                            "name": "should_show_custom_notifications",
+                            "value": "false"
+                        }
+                    ],
+                    "feature_association": {
+                        "enable_feature": ["AdNotifications"],
+                        "disable_feature": []
+                    }
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 0
+                }
+            ],
+            "filter": {
+                "channel": ["BETA", "RELEASE"],
+                "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
+            }
+        },
+        {
+            "name": "BraveAds.AdNotificationsEnabledOnUnknownAndNightlyChannelStudy",
+            "experiments": [
+                {
+                    "name": "AdNotifications",
+                    "probability_weight": 40,
+                    "parameters": [
+                        {
+                            "name": "should_show_custom_notifications",
+                            "value": "false"
+                        }
+                    ],
+                    "feature_association": {
+                        "enable_feature": ["AdNotifications"],
+                        "disable_feature": []
+                    }
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 60
+                }
+            ],
+            "filter": {
+                "channel": ["UNKNOWN", "NIGHTLY"],
```

Comment:
do we have some undocumented use for `UNKNOWN`?

looking at https://github.com/brave/brave-variations/blob/main/seed/proto/study.proto#L205-L207
it shouldn't be used in the seed file.
```
    // UNKNOWN value is defined here for the benefit of code using this enum
    // type, but is not actually meant to be encoded in the protobuf.
    UNKNOWN = -1;
```

Reply:
@goodov `UKNOWN` targets dev channel. Dev builds should use CLI overrides for testing, not sure if there are other dev channel browsers in the wilde that we need to target @tmancey?

Reply:
got it, but the channel can be overridden using CLI, so I'm not sure why we should do this in stage/prod branches at all.

Reply:
good point, I probably wrote the script before I knew about the CLI args so if @tmancey or others don't have any other reasons for supporting it we can remove `UNKNOWN` from supported channels

Reply:
Removed `UNKNOWN`